src/node_perf.cc

```diff
@@ -366,6 +366,24 @@ void Timerify(const FunctionCallbackInfo<Value>& args) {
   args.GetReturnValue().Set(wrap);
 }
 
+// Notify a custom PerformanceEntry to observers
+void Notify(const FunctionCallbackInfo<Value>& args) {
+  Environment* env = Environment::GetCurrent(args);
+  Utf8Value type(env->isolate(), args[0]);
+  Local<Value> entry = args[1];
+  PerformanceEntryType entryType = ToPerformanceEntryTypeEnum(*type);
```

Comment:
tiny nit:

```suggestion
  PerformanceEntryType entry_type = ToPerformanceEntryTypeEnum(*type);
```

(`lower_snake_case` for local variables)

Reply:
Done